ldml/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: LDML
+description: SIL Keyman is adding support for the LDML Keyboard specification.
+---
+
+## LDML Support in Keyman
+
+SIL Keyman is adding support for the LDML Keyboard specification.
+
+### Introducing the LDML Keyboard specification
+
+LDML is the Locale Data Markup Language. It it an XML-based format specified as the Unicode Consortium’s specification UTS#35, maintained by the Common Locale Data Repository (CLDR) Technical Commitee. Part 7 of UTS#35 pertains to keyboard layouts. This specification is currently being rewritten and further developed by the [CLDR Keyboard Subcommittee](https://cldr.unicode.org/index/keyboard-workgroup).
+
+Like the [Keyman Keyboard Language](https://help.keyman.com/developer/language/) (`.kmn`) format, the LDML Keyboard format defines how keystrokes are interpreted and how keys are laid out in a virtual keyboard.  The LDML format has been defined from the beginning to become a cross-platform industry standard, with active participation from several Unicode member organizations including SIL International.  The intention is to provide a draft specification as a technical preview for public review in mid-2023, with a first release later in 2023.
+
+### Keyman and LDML
+
+As was announced in the [March 2022](https://blog.keyman.com/2022/03/keyman-roadmap-march-2022/) [February 2023](https://blog.keyman.com/2023/02/keyman-roadmap-february-2023/) roadmaps, the Keyman team has been working in anticipation of LDML support in Keyman and Keyman Developer.  We plan to provide initial support for the LDML technical preview version in desktop devices in the Keyman 17.0 release and some supporting tools, with Keyman Developer support and touch platforms coming in version 18.0.
+
+The LDML support is being developed in our cross-platform Keyman Core component, which will enable a consistent feature set across all platforms including web by version 19.0, with a common code base.
+
+![Movie: Keyman on Linux, typing using an LDML keyboard](/cdn/dev/img/ldml-keyman-linux.gif)
+
+## Frequently Asked Questions for Keyman LDML Support
+
+
+- *How will the LDML format be supported with Keyman?*
+
+  From a technical perspective, the LDML format will be a newly supported input to the Keyman Command Line Compiler. The XML file will be compiled into a <a href="https://help.keyman.com/developer/16.0/reference/file-types/kmx">KMX compiled keyboard file</a>. The .kmx file will be read and used by the Keyman Core component, starting in version 17.0</dd>
```

Comment:
> The .kmx file will be read and used by the Keyman Core component, starting in version 17.0

This is misleading - Core already reads and uses .kmx files since the beginning of Core.

Reply:
Thanks. I'll update it